server/routes/index.js

```diff
@@ -66,70 +67,86 @@ module.exports = function Index({
     next()
   })
 
-  const reportPath = formName => `/report/:bookingId/${formName}`
+  // All users
+  {
```

Comment:
Consider extracting to functions with meaningful names (to eliminate comments)

Reply:
I was torn.. it's top level of the module so it looked a bit weird..

I'll address this in a follow up